core/src/gl/uniformEntries.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+namespace Tangram {
+
+namespace UniformEntries {
+
+typedef uint16_t EntryId;
+
+struct UniformEntry {
```

Comment:
Ah yeah I can see how debugging might be a pain. If we make our "built-in" uniforms an enum then those will be named in a debugger - generated uniforms would still be hard to debug.